src/logger.js

```diff
@@ -0,0 +1,13 @@
+// By default, do nothing
+exports.log = function () {};
+
+exports.setLogger = function setLogger(fn) {
+  exports.enabled = true;
```

Comment:
Instead of exporting this and add the check everywhere, we can just make the check on the log function itself (at line 10).